### ISSUE TITLE
Coding errors in org.h2.server.web.WebServer

### ISSUE THREAD

**Issue by GoogleCodeExporter** [sagarswathi/h2database#244]
```
WebServer.java:
line 585: 
    prop.setProperty("webAllowOthers", "" + SortedProperties.getBooleanProperty(old, "webAllowOthers", ssl));
line 586: 
    prop.setProperty("webSSL", "" + SortedProperties.getBooleanProperty(old, "webSSL", allowOthers));

   The argument "ssl" and "allowOthers" should exchange position?
```

Original issue reported on code.google.com by `live.t...@gmail.com` on 27 Oct 2010 at 7:08

**Comment by GoogleCodeExporter** [comment]
```
Thanks a lot! This will be fixed in the next release.
```

Original comment by `thomas.t...@gmail.com` on 28 Oct 2010 at 6:34
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
Fixed in version 1.2.145.
```

Original comment by `thomas.t...@gmail.com` on 2 Nov 2010 at 7:30
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
Thanks, well done!
```

Original comment by `live.t...@gmail.com` on 3 Nov 2010 at 2:45
